src/rsz/test/hi_fanout.tcl

```diff
@@ -185,7 +185,7 @@ proc write_clone_test_def1 { filename clone_gate fanout
     puts $stream "UNITS DISTANCE MICRONS $dbu ;"
     # Write out the die area 
     write_diearea $stream $fanout $load_spacing
-    puts "$filename $clone_gate $fanout $drvr_inst $drvr_cell $drvr_clk $drvr_out "
+    puts "./[string trimleft [string map [list [pwd]/ ""] [file normalize  $filename]] "/"] $clone_gate $fanout $drvr_inst $drvr_cell $drvr_clk $drvr_out "
```

Comment:
These two puts seem like debug code as everything else writes to $stream.  I would just remove them.

Reply:
Separate concern, separate PR... https://github.com/The-OpenROAD-Project/OpenROAD/pull/5435